packages/gatsby-core-utils/src/index.ts

```diff
@@ -21,5 +21,7 @@ export { createFilePath } from "./filename-utils"
 export { readConfigFile, getConfigPath } from "./utils"
 export { lock } from "./lock"
 export { murmurhash } from "./murmurhash"
+export * from "./hash"
+export { md5File } from "./md5-file"
```

Comment:
We should add a note about these to the README as well since we're documenting the exports there